src/Domain/Creators/Factory3D.hpp

```diff
@@ -21,8 +21,8 @@ template <>
 struct domain_creators<3> {
   using type =
       tmpl::list<domain::creators::AlignedLattice<3>,
-                 domain::creators::BinaryCompactObject, domain::creators::Brick,
-                 domain::creators::Cylinder,
+                 domain::creators::BinaryCompactObject<false>,
+                 domain::creators::Brick, domain::creators::Cylinder,
```

Comment:
In the BBH executable, you also need to have `BCO<false>` since we use a different list there